index.html

```diff
@@ -305,23 +305,24 @@ <h3>
           "RESOURCE-TIMING-2#dom-performanceresourcetiming">PerformanceResourceTiming</dfn></code>
           interface:
         </p>
-        <ul data-dfn-for="PerformanceResourceTiming">
-          <li>The <dfn>initiatorType</dfn> attribute MUST return the
-          {{DOMString}} "<code>navigation</code>".
+        <ul>
+          <li>The <dfn for="PerformanceNavigationTiming" id='dom-PerformanceNavigationTiming-workerStart'><code>
+            workerStart</code></dfn> getter steps are to perform the following steps:
```

Comment:
I think we should just go with: 

```suggestion
          <li>The `workerStart` getter steps are to perform the following steps:
```

We can clean up the linking after.